apps/src/templates/instructions/ChatBubble.jsx

```diff
@@ -51,6 +52,8 @@ var audioStyle = {
   },
 };
 
+const MINECRAFT_COLOR = '#3B3B3B';
```

Comment:
I see that we have `$very-dark-gray` in `apps/style/craft/style.scss`. There's no easy way to import that file in this one. Hovewer, I guess we can rename it something like `MINECRAFT_VERY-DARK-GRAY-COLOR` for consistency.

Reply:
Will do - thanks!